Libraries/Components/Pressable/useAndroidRippleForView.js

```diff
@@ -90,5 +99,5 @@ export default function useAndroidRippleForView(
       };
     }
     return null;
-  }, [rippleColor, viewRef]);
```

Comment:
I think rippleconfig is likely to be different every render. Maybe spread the options out first so it does value comparisons on the specific options.

Reply:
ah 🙈 I explicitly enumerated all the `rippleConfig` properties. I also did a small refactor of how `useMemo` is used (no need to memoize the returned `null` in case ripple is not supported / configured. It makes the diff a little bigger but I just switched the ifs.

Reply:
reverted this diff